index.js

```diff
@@ -0,0 +1,76 @@
+document.addEventListener("DOMContentLoaded",()=>{
+
+fetch("https://ghibliapi.herokuapp.com/films")
+.then(response => response.json())
+.then(response => {
+  getFilms(response)
+})
+
+let select = document.querySelector("select")
+select.addEventListener("change",(e)=>{
+  let words = document.querySelector("#words")
+  words.value = ""
+  let info = document.querySelector("#info")
+  let index = arrjson[e.target.selectedIndex]
+  fetch(`https://ghibliapi.herokuapp.com/films/${index}`)
+.then(response => response.json())
+.then(response => {
+  while(info.firstChild){
+    info.removeChild(info.firstChild);
+  }
+  let title = document.createElement("h3")
+  let releaseYear = document.createElement("p")
+  let description = document.createElement("p")
+  title.innerText = response.title;
+  releaseYear.innerText = response.release_date
+  description.innerText = response.description
+  if(index === null){
+    title.innerText = "";
+    releaseYear.innerText = ""
+    description.innerText = ""
+  }
+info.appendChild(title)
+info.appendChild(releaseYear)
+info.appendChild(description)
```

Comment:
Code indentation, spacing and nesting was poorly done, which makes hard to parse through the code to see what belongs where and what happens first.

Reply:
If your are using vs code try `right click` then `format document` or just `option+shift+f`